include/zephyr/bluetooth/audio/has.h

```diff
@@ -45,7 +45,7 @@ struct bt_has;
 
 /** Hearing Aid device type */
 enum bt_has_hearing_aid_type {
-	BT_HAS_HEARING_AID_TYPE_BINAURAL,
+	BT_HAS_HEARING_AID_TYPE_BINAURAL = 0,
 	BT_HAS_HEARING_AID_TYPE_MONAURAL,
 	BT_HAS_HEARING_AID_TYPE_BANDED,
```

Comment:
> We could consider making them with 0x01, 0x02 and 0x03 as those are the literal values that the HAS spec defines, and that we can be more sure that they are always correct

I would then rather use the BIT notation that is also used just below  0, BIT(0), BIT(1)

Reply:
I understand why you see that as a good option, since it's what the spec uses. My opinion is still that the way the spec presents it, using bits and not values, is a poor choice as it is not a bitfield. For that same reason, I believe that 

```c
	BT_HAS_HEARING_AID_TYPE_BINAURAL = 0x00,
	BT_HAS_HEARING_AID_TYPE_MONAURAL = 0x01,
	BT_HAS_HEARING_AID_TYPE_BANDED = 0x02,
```

is the best option, to avoid any confusion. 

@MariuszSkamra any input?

Reply:
@Thalley - I see your point.  I'll change it to hex values for clarity.

Reply:
> It is a bitfield actually, as the Hearing Aid type are two LSB in Hearing Aid Features characteristic. I won't object if you leave it as hex, it's just a matter of preference

True, saying it isn't a bitfield is wrong on my side. It is a bitfield, but where the bits are exclusive, i.e. you cannot set more than one bit. The "cannot set more than one bit" is what I want to empathize, which I am worried that using the `BIT(x)` macro won't. 

We can keep the `BIT(x)` to stay more "true" to the spec, if we add proper documentation of this, and ensuring that no more than 1 bit is set. 

`0xb11` is currently RFU, but if they add another type `BT_HAS_HEARING_AID_TYPE_NEW`, which would have the value `0b11`, then suddenly (if using `BIT(x)`) we might indicate that `BT_HAS_HEARING_AID_TYPE_MONAURAL | BT_HAS_HEARING_AID_TYPE_MONAURAL == BT_HAS_HEARING_AID_TYPE_NEW`, because the `OR`/`|` is typically something you can do with values defined with `BIT(x)`. 

TL;DR: I am OK with either hex values (0x00) or using the `BIT(x)` (although we cannot denote `BT_HAS_HEARING_AID_TYPE_BINAURAL` with `BIT(x)`) if it gets properly documented, though I still prefer the hex values.